pandas/core/internals/blocks.py

```diff
@@ -552,10 +552,10 @@ def f(m, v, i):
 
         return self.split_and_operate(None, f, False)
 
-    def astype(self, dtype, copy=False, errors="raise", values=None, **kwargs):
-        return self._astype(dtype, copy=copy, errors=errors, values=values, **kwargs)
```

Comment:
For everyone's edification, why is this undesired exactly?

I don't object to signature simplifications of course, but I'm curious why it was so undesired in the first place.

Reply:
I'm having trouble tracking down the relevant conversation, but the long and short of is: i looked up in the git blame where this was introduced, asked the author (jreback) if it was necessary, and it was confirmed that it isn't.